binding/go/porcupine_test.go

```diff
@@ -13,28 +13,61 @@ package porcupine
 import (
 	"encoding/binary"
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"math"
 	"os"
 	"path/filepath"
 	"testing"
+	"reflect"
 )
 
-var pvTestAccessKey string
+var (
+	pvTestAccessKey string
+	p               Porcupine	
```

Comment:
in the previous version, `p` was local in a function and it made sense to have a short name for it. Now that it has a larger scope we prob should give it a better name. `porcupine` ...